tests/apps/loqus/test_loqusdbapi.py

```diff
@@ -0,0 +1,70 @@
+import subprocess
+
+from cg.apps.loqus import LoqusdbAPI
+
+def test_instatiate(loqus_config):
+    """Test to instantiate a loqusdb api"""
+    ## GIVEN a loqusdb api with some configs
+
+    ## WHEN instantiating a loqusdb api
+    loqusdb = LoqusdbAPI(loqus_config)
+
+    ## THEN assert that the adapter was properly instantiated
+    assert loqusdb.uri == loqus_config['loqusdb']['database']
+    assert loqusdb.db_name == loqus_config['loqusdb']['database_name']
+    assert loqusdb.loqusdb_binary == loqus_config['loqusdb']['binary']
+
+def test_get_case(loqusdbapi, mocker):
+    """Test to get a case via the api"""
+    ## GIVEN a loqusdb api
+    case_id = 'a_case'
+    ## WHEN fetching a case with the adapter
+    mocker.patch.object(subprocess, 'check_output')
+    subprocess.check_output.return_value = b'[{"_id": "a_case"}]'
+    case_obj = loqusdbapi.get_case(case_id)
+    ## THEN assert that the correct case id is returned
+    assert case_obj['_id'] == case_id
+
+def test_get_case_non_existing(loqusdbapi, mocker):
+    """Test to get a case via the api"""
+    ## GIVEN a loqusdb api
+    case_id = 'a_case'
+    ## WHEN fetching a case with the adapter
+    mocker.patch.object(subprocess, 'check_output')
+    subprocess.check_output.side_effect = subprocess.CalledProcessError(1,'error')
+    case_obj = loqusdbapi.get_case(case_id)
+
+    ## THEN assert None is returned
+    assert case_obj == None
+
+def test_load(loqusdbapi, mocker):
+    """Test to load a case"""
+    ## GIVEN a loqusdb api and some info about a case
+    family_id = 'test'
+    ped_path = 'a ped path'
+    vcf_path = 'a vcf path'
+
+    ## WHEN loading the case to loqus with the api
+    mocker.patch.object(subprocess, 'check_output')
+    subprocess.check_output.return_value = (b'2018-11-29 08:41:38 130-229-8-20-dhcp.local '
```

Comment:
It is a good thing that we test stuff, I'm a bit confused about what we do at this point. If we need this type of hard coded values, could we hide it in the conftest if it is not relevant for understanding the purpose and function of the test?

Reply:
When are you in, it would be nice to talk a bit about this?

Reply:
Sure, I'll hope to be back om Friday